src/libraries/Native/Unix/System.Native/pal_datetime.c

```diff
@@ -39,3 +41,20 @@ int64_t SystemNative_GetSystemTimeAsTicks()
     // in failure we return 00:00 01 January 1970 UTC (Unix epoch)
     return 0;
 }
+
+char* SystemNative_GetDefaultTimeZone()
+{
+#if defined(TARGET_ANDROID)
```

Comment:
I think you need to condition the whole function so that only android has it.

Reply:
Thanks, updated!